### ISSUE TITLE
Feature request: Handle newlines on Quizlet import

### ISSUE THREAD

**Issue by kewbish** [Yvee1/hascard#17]
First off - thanks for creating this clear + simple TUI. It's been super great so far, but I'd like to suggest a feature request for Quizlet importing. 

This is related to #12 - your comment there stated:
> The exporting from Quizlet should be done with the default options, so a tab between term and definition, and a new line between rows.

Some sets on Quizlet have flashcard definitions with multiple lines, where the question and answer are copied into a plain text file like so:
```text
Question?    - answer part 1
- answer part 2
```
When I try to run `hascard import`, I get `Failed the conversion.`

Currently, I can run a couple regex commands on a Quizlet set exported with a custom delimiter to get it into the hascard format. However, is it possible to add support for multiple newlines between flashcards? This way, Quizlet's export feature with the custom delimiter of `\n\n` can be used to directly import.

**Comment by Yvee1** [comment]
Hi! I see the problem, thanks for pointing it out. 

I'll add command-line options for specifying the delimiters, then you will be able to import files that use the `\n\n` delimiter like you mentioned. This also makes the import process more versatile in general, so I think it's a nice addition.

I'll let you know once it's implemented!

**Comment by Yvee1** [comment]
It is implemented in release 0.5.0.3 :). You should be able to run, for example,
```
hascard import input.txt output.txt --row-delimiter "\n\n" -t def
```
If `input.txt` is the example plain text file you gave, then this yields the following `output.txt` file:
```
# Question?
- answer part 1
- answer part 2
```
(Actually, it doesn't exactly because the example you gave uses 4 spaces instead of tabs; but for that you could change the default "term-def-delimiter" from tabs to four spaces, as below)
```
hascard import input.txt output.txt --row-delimiter "\n\n" --term-def-delimiter "    " -t def
```
I'm having some struggles with macos in Github Actions, so there is no release for that atm.

**Comment by kewbish** [comment]
Thank you so much - works perfectly! Appreciate your quick response and this amazing TUI :)
